app/controllers/api/projects.php

```diff
@@ -1340,7 +1340,7 @@
         $target = new Domain(App::getEnv('_APP_DOMAIN_TARGET', ''));
 
         if (!$target->isKnown() || $target->isTest()) {
-            throw new Exception(Exception::GENERAL_SERVER_ERROR, 'Unreachable CNAME target (' . $target->get() . '), please use a domain with a public suffix.');
+            throw new Exception(Exception::DOMAIN_VARIABLE_INCORRECT, 'Unreachable CNAME target (' . $target->get() . '). Please check the _APP_DOMAIN_TARGET environment variable of your Appwrite server.');
```

Comment:
@singhbhaskar this is actually a 500 error and not supposed to be exposed on UI in production mode. These errors are supposed to be logged into logging services like sentry where you can find the entire stack trace of this exception

Reply:
So this PR may not be relevant